src/wirecraft/__main__.py

```diff
@@ -23,7 +25,8 @@ def init_game():
             resolution = Resolution(settings["resolution"]["width"], settings["resolution"]["height"])
     except FileNotFoundError:
         resolution = Resolution(1920, 1080)
-
+    with contextlib.suppress(FileNotFoundError):
+        pathlib.Path("database.db").unlink(missing_ok=True)
```

Comment:
```suggestion
    pathlib.Path("database.db").unlink(missing_ok=True)
```

You already use missing_ok=True